core/graph/src/test/java/uk/gov/gchq/gaffer/graph/GraphTest.java

```diff
@@ -166,73 +167,41 @@ public void shouldConstructGraphFromSchemaModules() {
         storeProperties.setStoreClass(TestStoreImpl.class.getName());
 
         final Schema schemaModule1 = new Schema.Builder()
-                .type(TestTypes.PROP_STRING, new TypeDefinition.Builder()
-                        .clazz(String.class)
-                        .build())
-                .type("vertex", new TypeDefinition.Builder()
-                        .clazz(String.class)
-                        .build())
-                .edge(TestGroups.EDGE, new SchemaEdgeDefinition.Builder()
-                        .property(TestPropertyNames.PROP_1, TestTypes.PROP_STRING)
-                        .aggregate(false)
-                        .source("vertex")
-                        .destination("vertex")
-                        .directed(DIRECTED_EITHER)
-                        .build())
+                .type(TestTypes.PROP_STRING, new TypeDefinition.Builder().clazz(String.class).build())
```

Comment:
There are a lot of changes to this file, most are whitespace from what I can tell. Please could you revert some of the whitespace changes so it's easier to review?

Reply:
Sorry about that - was trying to setup my save actions to work with the Gaffer checkstyle and it looks like it's started merging lines (this also seems to have caused the measured test coverage % to drop) - I'll sort that out